tests/.pest/snapshots/Countries/PolandTest/it_can_calculate_polish_holidays.snap

```diff
@@ -0,0 +1,54 @@
+[
+    {
+        "name": "Nowy Rok",
+        "date": "2024-01-01"
+    },
+    {
+        "name": "Święto Trzech Króli",
```

Comment:
By default JOSN encodes UTF-8 to escaped unicode points, so to get the test passing it should be 
```suggestion
        "name": "\u015awi\u0119to Trzech Kr\u00f3li",
```
This should be corrected for all names with polish-specific characters